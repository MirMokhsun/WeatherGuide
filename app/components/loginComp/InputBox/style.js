import {StyleSheet, Dimensions} from 'react-native';
import colors from '../../../services/constans/colors';

const width = Math.round(Dimensions.get('window').width);

export const styles = StyleSheet.create({
  InputsView: {
    flex: 1,
    position: 'relative',
    marginTop: '5%',
  },
  inputView: {
    height: 20,
  },
  inputView2: {
    height: 20,
    marginTop: 60,
  },
});
