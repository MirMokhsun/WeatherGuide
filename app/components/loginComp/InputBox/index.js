import React from 'react';
import { View } from 'react-native';
import { styles } from '../style';
import { textInput } from '../textInput'


const InputBox = () => {
    return (
        <View style={styles.InputsView}>
            <View style={styles.VB1}>
                {textInput('Enter your email', null, 'login', null)}
            </View>
            <View style={styles.VB2}>
                {textInput('Enter your password', null, 'textbox-password', null)}
            </View>
        </View>
    )
}

export default InputBox