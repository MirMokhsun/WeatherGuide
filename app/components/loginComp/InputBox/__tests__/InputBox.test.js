import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import InputBox from '../'

describe('InputBox', () => {
    test('InputBox snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<InputBox />);
        expect(snapshot).toMatchSnapshot();
    });
});