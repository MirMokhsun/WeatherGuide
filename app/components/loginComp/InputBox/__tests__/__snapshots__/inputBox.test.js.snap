// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputBox InputBox snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "marginTop": "5%",
      "position": "relative",
    }
  }
>
  <View
    style={
      Object {
        "height": 20,
      }
    }
  >
    <View
      style={
        Object {
          "borderColor": "gray",
          "borderRadius": 25,
          "borderWidth": 2,
          "flexDirection": "row",
          "height": 60,
          "width": 500,
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Enter your email"
        rejectResponderTermination={true}
        style={
          Object {
            "fontSize": 16,
            "height": 60,
            "paddingLeft": 50,
            "width": "100%",
          }
        }
        underlineColorAndroid="transparent"
        value={null}
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "height": "100%",
            "justifyContent": "center",
            "position": "absolute",
            "width": 60,
          }
        }
      >
        <Icon
          allowFontScaling={false}
          color="rgba(33, 57, 156,0.6)"
          name="login"
          size={25}
        />
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "height": 20,
        "marginTop": 60,
      }
    }
  >
    <View
      style={
        Object {
          "borderColor": "gray",
          "borderRadius": 25,
          "borderWidth": 2,
          "flexDirection": "row",
          "height": 60,
          "width": 500,
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Enter your password"
        rejectResponderTermination={true}
        style={
          Object {
            "fontSize": 16,
            "height": 60,
            "paddingLeft": 50,
            "width": "100%",
          }
        }
        underlineColorAndroid="transparent"
        value={null}
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "height": "100%",
            "justifyContent": "center",
            "position": "absolute",
            "width": 60,
          }
        }
      >
        <Icon
          allowFontScaling={false}
          color="rgba(33, 57, 156,0.6)"
          name="textbox-password"
          size={25}
        />
      </View>
    </View>
  </View>
</View>
`;
